tests/contrib/eventinfo/test_event_types.py

```diff
@@ -53,3 +53,15 @@ def test__event_types__scheduled(event_scheduled):
 
     expected_keys = ['@iopipe/event-info.eventType'] + ['@iopipe/event-info.scheduled.%s' % key for key in event.keys]
     assert list(event_info.keys()).sort() == expected_keys.sort()
+
+
+def test__event_types__serverless_lambda(event_serverless_lambda):
+    event = et.ServerlessLambda(event_serverless_lambda)
+    assert event.has_required_keys() is True
+
+    event_info = event.collect()
+    assert event_info != {}
+
+    expected_keys = ['@iopipe/event-info.eventType'
```

Comment:
Coverage for `eventType.source`? https://github.com/iopipe/iopipe-js-event-info/commit/2033c3e66976de1855a5746f33c745bf76492b03#diff-ee6f6f22184012610d36277ea8371b4bR87 That is, can we add a test asserting that key/val, for consistency?

Reply:
I don't understand the question. The test asserts that the event type produces the expected result (`new_key`), which is the api gateway keys. This is effectively testing the mapping.

Reply:
I added an assertion to do what I think you're describing.